contracts/TopShot.cdc

```diff
@@ -47,62 +47,65 @@ import FungibleToken from 0xFUNGIBLETOKENADDRESS
 import NonFungibleToken from 0xNFTADDRESS
 import MetadataViews from 0xMETADATAVIEWSADDRESS
 import TopShotLocking from 0xTOPSHOTLOCKINGADDRESS
+import ViewResolver   from 0xVIEWRESOLVERADDRESS
 
-pub contract TopShot: NonFungibleToken {
+access(all) contract TopShot: NonFungibleToken, ViewResolver {
     // -----------------------------------------------------------------------
     // TopShot deployment variables
     // -----------------------------------------------------------------------
 
+    access(all) entitlement NFTMinter
+
     // The network the contract is deployed on
-    pub fun Network() : String { return ${NETWORK} }
+    view access(all) fun Network() : String { return ${NETWORK} }
```

Comment:
This is just a style thing, but the common place for putting `view` in a function declaration is after the access specification, so `access(all) view fun Network...`